python/requirements/ml/core-requirements.txt

```diff
@@ -4,7 +4,7 @@ mlflow==2.9.2
 wandb==0.17.0
 
 # ML training frameworks
-xgboost==1.7.6
+xgboost==2.1.0
```

Comment:
For my own understanding, this requirements file shows the dependency to build our xxx-ml image, is that correct?

Reply:
Yeah, which is used in CI